packages/apollo-gateway/src/index.ts

```diff
@@ -32,26 +33,47 @@ export interface GraphQLService {
 
 export type ServiceEndpointDefinition = Pick<ServiceDefinition, 'name' | 'url'>;
 
-export interface GatewayConfigBase {
+interface GatewayConfigBase {
   debug?: boolean;
   // TODO: expose the query plan in a more flexible JSON format in the future
   // and remove this config option in favor of `exposeQueryPlan`. Playground
   // should cutover to use the new option when it's built.
   __exposeQueryPlanExperimental?: boolean;
   buildService?: (definition: ServiceEndpointDefinition) => GraphQLDataSource;
-  serviceList?: ServiceEndpointDefinition[];
 }
 
+export interface ConcreteGatewayConfig extends GatewayConfigBase {
```

Comment:
`Concrete` and `Hosted` don't feel like the right naming here? `RemoteGatewayConfig` and `ManagedGatewayConfig` instead?

Reply:
Changed